### PR TITLE
VDR-19 VDR-17 Update issuer data permissions + certifier check for schemas

### PR DESCRIPTION
**Pull request by vivi-tz**

### CODE REVIEW

File: registryLogic.py [hunk 1]
```diff
@@ -225,6 +246,10 @@ def add_issuer(self, parameters):
         sp.set_type(parameters.issuer_did, sp.TString)
         sp.set_type(parameters.issuer_data, sp.TString)
 
+        # Check if issuer does not exist, does not allow add call otherwise
+        # This is to avoid update from unauthorized
+        sp.verify(~self.check_issuer_exists(parameters.issuer_did), message="Issuer did already exists")
```
Comment (yurivis) [inline registryLogic.py]: Any reason for the `~` before `self`?

**Review by yurivis — COMMENTED** [review]
We can follow PEP 8 for styling, since this is still technically Python we're writing. In there there is a [recommendation](https://peps.python.org/pep-0008/#other-recommendations) about spaces and operators. We disregard them in some cases. For example line 160, of `registryLogic.py` where `owner_address=owner_address` should be `owner_address = owner_address`

Otherwise I don't see any technical issues

**Comment by vivi-tz** [comment]
> We can follow PEP 8 for styling, since this is still technically Python we're writing. In there there is a [recommendation](https://peps.python.org/pep-0008/#other-recommendations) about spaces and operators. We disregard them in some cases. For example line 160, of `registryLogic.py` where `owner_address=owner_address` should be `owner_address = owner_address`
> 
> Otherwise I don't see any technical issues

ok, sure!  what do you recommend for formatting? A specific IDE? 
I was using PyCharm but when I formatted it it changed it drastically

File: registryLogic.py (hunk 1 above)
Comment (vivi-tz) [reply registryLogic.py]: > Any reason for the `~` before `self`?

it's the logical not

**Comment by yurivis** [comment]
Finding no solution about auto-formatting, because SmartPy fails fail some PEP 8 recommendations anyway, we fixed the formatting by hand.

**Pull request merged**